packages/ruleset/src/functions/string-boundary.js

```diff
@@ -1,6 +1,5 @@
-module.exports = function(schema, _opts, paths) {
-  const rootPath = paths.target !== void 0 ? paths.target : paths.given;
```

Comment:
This line is no longer relevant with the new spectral - it is removed from a number of files